src/test/java/com/ita/if103java/ims/contoller/AssociateControllerTest.java

```diff
@@ -1,7 +1,8 @@
-package com.ita.if103java.ims.controller;
+package com.ita.if103java.ims.contoller;
```

Comment:
Misstype

Reply:
There were two packages but unfortunately, I've deleted the right package.